src/components/Anchor/Anchor.tsx

```diff
@@ -12,11 +12,11 @@ export const qaIdByDefault = 'qa-anchor';
 
 export interface AnchorProps extends ClassNameProps {
     id: string;
-    dataQa?: string;
+    qa?: string;
```

Comment:
It looks like... but it does not. Actually, this props was added for tests a few commits ago. When I started to develop shared test code found out that I had named this prop not like same props in other components.
I think it will no break any of our projects